metrics/Cumulative.md

```diff
@@ -0,0 +1,51 @@
+# Cumulative Overview
+A `Cumulative` is used to record aggregated metrics that represents a single numerical value accumulated over a time interval. The value can only increase or be reset to zero on restart or reset the event. The recorded values are always >= 0. Typical examples of counters would be the number of elements added to a queue, total cpu usage, tasks completed, cache misses, etc.
```

Comment:
Typical examples of counters -> Typical examples of cumulative

Reply:
Done